fastify.js

```diff
@@ -337,6 +338,8 @@ function build (options) {
   }
 
   function inject (opts, cb) {
+    if (!shot) return cb(new Error('"shot" library is not installed: "npm install shot --save-dev"'))
```

Comment:
Probably here we want to *throw* an error, because the user will not expect an error in the callback you are using.

```js
fastify.inject({
  // option object
}, response => {
  // doing something with the response
})
```